packages/@okta/vuepress-theme-prose/assets/css/okta/pages/multi-tenancy.scss

```diff
@@ -0,0 +1,54 @@
+.multi-tenancy {
```

Comment:
I think these styles should be global, and not scoped to anything specific to this article. But, I'll defer to engineering.

Reply:
I'll continue coordinating with Design(Christine) and the dev docs engineering team (Andrii, Shaw, etc) and move these changes out when they become global.